IdentityCore/src/oauth2/token/MSIDAccessToken.m

```diff
@@ -108,6 +108,7 @@ - (instancetype)initWithTokenCacheItem:(MSIDCredentialCacheItem *)tokenCacheItem
     if (self)
     {
         _expiresOn = tokenCacheItem.expiresOn;
+        _extendedExpireTime = tokenCacheItem.extendedExpiresOn;
```

Comment:
nit: should we call it consistently extendedExpiresOn?

Reply:
Agreed, in a separate PR